src/core/include/units/quantity.h

```diff
@@ -392,17 +392,17 @@ class quantity {
     return ret(v * q.number());
   }
 
-  template<typename Value>
-    requires(!Quantity<Value>) && (invoke_result_convertible_to_<rep, std::divides<>, rep, const Value&>)
+  template<Representation Value>
+    requires(invoke_result_convertible_to_<rep, std::divides<>, rep, const Value&>)
```

Comment:
```suggestion
    requires invoke_result_convertible_to_<rep, std::divides<>, rep, const Value&>
```

Reply:
@JohelEGP This may format strangely with clang-format. I had to put many extraneous parenthesis for clang-format to not go nuts 😉

Reply:
At least this doesn't change with Clang 16.

Reply:
Sorry I didn't respond faster. I wasn't sure about this proposition as it suggested the application of similar changes for the other operators. 

As I didn't want to review the `requires` syntax convention, I was a bit hesitant to apply the suggestions.

Reply:
I think I remember now. Clang-Format didn't have good support for formatting *requires-clause*s, but now it does.

Reply:
I will revisit this in the V2 branch. But I prefer to stick to clang-format-15 as this is the first one that somehow supports our codebase but the last one released. I do not want to force people to install clang-format-16 for now.